frontend/src/components/Wallet/LoginModal.tsx

```diff
@@ -71,22 +78,30 @@ const LoginModal: React.FC<IProps> = ({ onLogin, ...rest }) => {
     // },
   ];
   return (
-    <Dialog style={{ maxWidth: 360 }} {...rest}>
+    <Dialog style={{ maxWidth: 360 }} title={'Connect through'} {...rest}>
       <Root>
         <Img height="60" width="60" src={logo} />
         <SizedBox height={4} />
         <Text fitContent size="medium">
           Connect wallet
         </Text>
         <SizedBox height={4} />
-        <Text fitContent type="secondary" weight={500} size="tiny">
+        <Text fitContent type="secondary" size="tiny">
           To start using Spark
         </Text>
         <SizedBox height={34} />
         {!isImportInputOpened ? (
           loginTypes.map(
-            (t, i) =>
-              t.isActive && <LoginType {...t} key={i} onClick={t.onClick} />
+            (t, i) => (
```

Comment:
please change to more readable code, ternary operators we use in simple checks only, other vise its baad